packages/dashboard/src/pages/Wallets/components/WalletStateBar/index.tsx

```diff
@@ -121,7 +133,9 @@ export const WalletStateBarUI: FC<WalletStateBarUIProps> = ({
                     <WalletIcon providerIcon={provider.icon} inverse size={38} />
                 </Stack>
                 <Box sx={{ userSelect: 'none' }}>
-                    <Box fontSize={16}>{wallet.name}</Box>
+                    <Box fontSize={16} display="flex" alignItems="center">
+                        {domain && Utils?.formatDomainName ? Utils.formatDomainName(domain) : wallet.name}
```

Comment:
To change the signature of `formatDomainName`  to:

```ts
function formatDomainName(domain?: string, fallback?: string): string {
    // code
}
```

```suggestion
                        {Utils.formatDomainName(domain, wallet.name)}
```